packages/extension-ui/src/partials/Header.tsx

```diff
@@ -3,20 +3,21 @@
 
 import type { ThemeProps } from '../types';
 
-import { faArrowLeft, faCog, faPlusCircle, faSearch } from '@fortawesome/free-solid-svg-icons';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import React, { useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
 import styled from 'styled-components';
 
-import logo from '../assets/pjs.svg';
-import { ActionContext } from '../components';
-import InputFilter from '../components/InputFilter';
-import Link from '../components/Link';
+import logo from '../assets/azeroLogo.svg';
+import helpIcon from '../assets/help.svg';
+import settingsIcon from '../assets/settings.svg';
+import { ActionContext, Link,Tooltip } from '../components';
 import useOutsideClick from '../hooks/useOutsideClick';
 import useTranslation from '../hooks/useTranslation';
 import { getConnectedTabsUrl } from '../messaging';
-import MenuAdd from './MenuAdd';
-import MenuSettings from './MenuSettings';
+// these will be reused in the future
```

Comment:
If you add todo: to this line a lot of tools can just display all todos in the code instead of just descriptive comments. It's a handy thing when going back and looking for stuff that was not yet done or missing :)